target/linux/lantiq/dts/RS353.dts

```diff
@@ -0,0 +1,318 @@
+/dts-v1/;
+
+/include/ "vr9.dtsi"
+
+/ {
+	model = "RS353 - Bintec Elmeg RS353";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200 init=/etc/preinit";
+		leds {
+                        running = &status;
+                        boot = &status;
+			wifi = &wlan;
+			usb2 = &usb; /* Front port is mapped to device 2 */
+			dsl = &dsl;
+                };
+
+	};
+
+	memory@0 {
+		reg = <0x0 0x8000000>;
+	};
+
+	sram@1F000000 {
+		cgu0: cgu@103000 {
+                        compatible = "lantiq,cgu-xway";
+                        reg = <0x103000 0x1000>;
+                        #clock-cells = <1>;
+			lantiq,phy-clk-src = <0x2>;
+		};
+	};
+
+	fpi@10000000 {
+		localbus@0 {
+			nor-boot@0 {
+				compatible = "lantiq,nor";
+				bank-width = <2>;
+				reg = <0 0x0 0x2000000>;
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				partitions {
+					compatible = "fixed-partitions";
+					#address-cells = <1>;
+					#size-cells = <1>;
+
+					boardconfig: partition@0 {
```

Comment:
I wrote a little mtdsplit driver to properly detect the root partition:

```
[    0.885699] lantiq nor flash device: 02000000 at 10000000
[    0.891126] ltq_nor: Found 1 x16 devices at 0x0 in 16-bit bank. Manufacturer ID 0x0000c2 Chip ID 0x00227e
[    0.900618] Amd/Fujitsu Extended Query Table at 0x0040
[    0.905801]   Amd/Fujitsu Extended Query version 1.3.
[    0.910811] number of CFI chips: 1
[    0.914272] 5 ofpart partitions found on MTD device ltq_nor
[    0.919769] Creating 5 MTD partitions on "ltq_nor":
[    0.924653] 0x000000000000-0x000000020000 : "logic_config"
[    0.932237] 0x000000020000-0x000000040000 : "logic_addr"
[    0.939870] 0x000000040000-0x000000060000 : "bootmonitor"
[    0.947334] 0x000000080000-0x0000000c0000 : "flash_config"
[    0.955004] 0x0000000c0000-0x000002000000 : "firmware"
[    0.980359] BOSS Header: TELDAT ClosedEyeVisual
[    0.984778] Looking for the rootfs partition...
[    0.989315] Offset: 00080000 Magic: D50D9943
[    0.993581] Offset: 000A0000 Magic: 878AB17A
[    0.997841] Offset: 000C0000 Magic: 334FE501
[    1.002111] Offset: 000E0000 Magic: E9B67767
[    1.006376] Offset: 00100000 Magic: F50E047C
[    1.010641] Offset: 00120000 Magic: DA1F0C69
[    1.014906] Offset: 00140000 Magic: D518D642
[    1.019171] Offset: 00160000 Magic: 7D77682D
[    1.023441] Offset: 00180000 Magic: 306DE38C
[    1.027712] Offset: 001A0000 Magic: E1D14362
[    1.031966] Offset: 001C0000 Magic: 73717368
[    1.036223] Rootfs found at offset 001c0000
[    1.040401] 1 bintec-fw partitions found on MTD device firmware
[    1.046348] 0x000000280000-0x000002000000 : "rootfs"
[    1.053188] mtd: device 5 (rootfs) set to be root filesystem
[    1.058823] 1 squashfs-split partitions found on MTD device rootfs
[    1.064943] 0x000000560000-0x000002000000 : "rootfs_data"
```

I have to align properly the rootfs on flash sector to make it working.